stripecardscan/detekt-baseline.xml

```diff
@@ -89,7 +89,6 @@
     <ID>TooGenericExceptionThrown:RetryTest.kt$RetryTest$throw RuntimeException()</ID>
     <ID>TooManyFunctions:ArrayExtensions.kt$com.stripe.android.stripecardscan.framework.util.ArrayExtensions.kt</ID>
     <ID>TooManyFunctions:CardImageVerificationActivity.kt$CardImageVerificationActivity : SimpleScanActivitySimpleScanStateful</ID>
-    <ID>TooManyFunctions:CardScanActivity.kt$CardScanActivity : ScanActivitySimpleScanStateful</ID>
```

Comment:
is this needed?

Reply:
Regenerated baselines that failed since some files changed path / location